data/blog/libtailscale.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: "libtailscale: how it's made"
+date: '2023-03-02'
+tags:
+  - 'tsnet'
+  - 'libtailscale'
+  - 'oss'
+authors: ['crawshaw']
+summary: 'libtailscale: a Tailscale C library'
```

Comment:
Something longer. We're currently using these as the excerpts on the homepage.

Suggest:
"libtailscale: a Tailscale C library. Using the Go toolchain to build a c-archive tsnet package that uses a userland network stack to implement TCP/IP inside a process."